inc/class-upgrade.php

```diff
@@ -1035,12 +1045,14 @@ private function migrate_woocommerce_archive_setting_to_shop_page() {
 	}
 
 	/**
-	 * Stores the initial `permalink_structure` option.
+	 * Stores the initial `permalink_structure` options.
 	 *
 	 * @return void
 	 */
-	public function set_permalink_structure_option_for_151() {
+	public function set_permalink_structure_options_for_151() {
 		WPSEO_Options::set( 'permalink_structure', get_option( 'permalink_structure' ) );
+		WPSEO_Options::set( 'category_base_url', get_option( 'category_base' ) );
+		WPSEO_Options::set( 'tag_base_url', get_option( 'tag_base' ) );
```

Comment:
Shouldn't these two lines be removed? This is the upgrade routine for 15.1 but we're targeting 15.3. Also, see previous comment.